examples/restaurant_demo/restaurant.py

```diff
@@ -0,0 +1,341 @@
+MAX_EUROS_SPENT_2 = 65
+
+LN_3 = math.log(3)
+
+
+class RestaurantStatistics:
+    def __init__(self, hours_filename, days_filename, epsilon=LN_3):
+
+        # Store the name of the csvs for daily and weekly restaurant data
+        self.hours_filename = hours_filename
+        self.days_filename = days_filename
+
+        # The privacy threshold, a number between 0 and 1
+        self._epsilon = epsilon
+
+        # The hourly and weekly data as pandas DataFrames
+        self._hour_visits = pd.read_csv(self.hours_filename, sep=",")
+        self._day_visits = pd.read_csv(self.days_filename, sep=",")
+
+    def count_visits_per_hour(self) -> tuple:
+        """Compute raw count of visits per hour of day and return two dictionaries 
+        that map an hour to the number of visits in that hour. 
+        
+        The first dictionary is the count calculation without any differential privacy, while the second one uses the PyDP library for a private calculation
+        """
+        non_private_counts = self.get_non_private_counts_per_hour()
+        private_counts = self.get_private_counts_per_hour()
+
+        return non_private_counts, private_counts
+
+    def count_visits_per_day(self) -> tuple:
+        """Compute raw count of visits per day of week and return two dictionaries 
+        that map a day to the number of visits in that day. 
+        
+        The first dictionary is the count calculation without any differential privacy, 
+        while the second one uses the PyDP library for a private calculation
+        """
+        non_private_counts = self.get_non_private_counts_per_day()
+        private_counts = self.get_private_counts_per_day()
+
+        return non_private_counts, private_counts
+
+    def sum_revenue_per_day(self) -> tuple:
+        """Compute revenue per day for the whole week. 
+        
+        The first dictionary is the count calculation without any differential privacy, 
+        while the second one uses the PyDP library for a private calculation
+        """
+        non_private_sum = self.get_non_private_sum_revenue()
+        private_sum = self.get_private_sum_revenue()
+
+        return non_private_sum, private_sum
+
+    def sum_revenue_per_day_with_preaggregation(self) -> tuple:
+        """Calculates revenue per day in the whole week while preagreggating the spending of each visitor before calculating the BoundedSum with PyDP.
+        
+        The first dictionary is the count calculation without any differential privacy, 
+        while the second one uses the PyDP library for a private calculation
+        """
+        non_private_sum = self.get_non_private_sum_revenue()
+        private_sum = self.get_private_sum_revenue_with_preaggregation()
+
+        return non_private_sum, private_sum
+
+    def get_non_private_counts_per_hour(self) -> dict:
+        """Compute the number of visits per hour without any differential privacy. Return a dictionary mapping hours to number of visits"""
```

Comment:
Here and elsewhere, it would look better if you lint it - cap at 80 characters per line

Reply:
Gotcha!